namespaces/cloud-platform-live-0.k8s.integration.dsd.io/formbuilder-services-staging/formbuilder-services-staging-admin-role.yaml

```diff
@@ -9,12 +9,18 @@ subjects:
   - kind: Group
     name: "github:form-builder"
     apiGroup: rbac.authorization.k8s.io
+  # both dev & staging Publisher service accounts
+  # are admins, so that they can both deploy to either
   - kind: ServiceAccount
     name: formbuilder-publisher-workers-dev
     namespace: formbuilder-platform-dev
   - kind: ServiceAccount
     name: formbuilder-publisher-workers-staging
     namespace: formbuilder-platform-staging
+  # ...but only the staging user datastore can read the staging
+  # service tokens
+  - kind: ServiceAccount
+    name: formbuilder-user-datastore-staging
```

Comment:
Here you've omitted the `namespace` attribute. I believe it will default to the current namespace but seeing that you're being explicit in dev you might want to add it here as well?

Reply:
good catch - now fixed

Reply:
oops, adding that commit means it's dismissed your review - could you re-approve please?